src/esp/bindings/GfxBindings.cpp

```diff
@@ -36,22 +36,25 @@ namespace gfx {
 void initGfxBindings(py::module& m) {
   // ==== RenderCamera ====
   py::class_<RenderCamera, Magnum::SceneGraph::PyFeature<RenderCamera>,
-             Magnum::SceneGraph::AbstractFeature3D,
+             Magnum::SceneGraph::Camera3D,
              Magnum::SceneGraph::PyFeatureHolder<RenderCamera>>(
       m, "Camera",
       R"(RenderCamera: The object of this class is a camera attached
       to the scene node for rendering.)")
       .def(py::init_alias<std::reference_wrapper<scene::SceneNode>,
                           const vec3f&, const vec3f&, const vec3f&>())
-      .def("setProjectionMatrix", &RenderCamera::setProjectionMatrix, R"(
-        Set this `Camera`'s projection matrix.
+      .def("setProjectionMatrix",
+           py::overload_cast<int, int, float, float, float>(
+               &RenderCamera::setProjectionMatrix),
+           R"(
+        Set this `Camera`'s projection matrix via view configuration.
       )",
            "width"_a, "height"_a, "znear"_a, "zfar"_a, "hfov"_a)
-      .def("getProjectionMatrix", &RenderCamera::getProjectionMatrix, R"(
-        Get this `Camera`'s projection matrix.
-      )")
-      .def("getCameraMatrix", &RenderCamera::getCameraMatrix, R"(
-        Get this `Camera`'s camera matrix.
+      .def("setProjectionMatrix",
```

Comment:
@mosra @erikwijmans :

Now the RenderCamera is inherited from MagnumCamera, which I believe magnum has the py bindings for this signature (setProjectionMatrix), which makes it unnecessary here.

Is that correct?

Reply:
Looks like there is just a property for the projection matrix itself: https://doc.magnum.graphics/python/magnum/scenegraph/Camera3D/#properties

So the get is redundant, but the setters aren't.

Reply:
@erikwijmans : We have two "setProjectionMatrix" here. The 1st one (line 46) definitely needs to be kept. I am talking about the 2nd one, which is just the original "setProjectionMatrix" defined in the base class.

I am asking if we need to keep this binding. My opinion is no, we do not need it since it is already done in Magnum python bindings.

Reply:
Ah, didn't look closely at the second one, yes, that is not needed.